tests/e2e/env-vars/chainsaw-test.yaml

```
@@ -0,0 +1,54 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: service-name
```

ops 

```suggestion
  name: env-vars
```

I can address this quickly, but let me know if you find a better name for it or any other adjustment